src/namespacedActions.js

```diff
@@ -0,0 +1,64 @@
+import Actions from './actions'
+
+export default class NamespacedActions extends Actions {
```

Comment:
What's the reasoning for not just making this the default `Actions`? The behaviour stays the same as before if you don't set a namespace, so I can't tell what the benefits are of having this as a separate class.